README.md

```diff
@@ -380,3 +380,23 @@ See [note](Statemind%20Lido%20V2%20Deployment%20Validation%2005-2023.pdf) conten
 - Informational Issues: 1 (1 Fixed)
 
 See [full report](Hexens%20Lido%20V2%20Oracle%20Security%20Review%20Report%2005-23.pdf) for more details.
+
+## 06-2023 Oxorio Lido V2 On-chain Audit
```

Comment:
IDK, should both reports have the '05-2023' date mark as they mention `May 10` on the title page?